core/src/include/ModelArray.hpp

```diff
@@ -113,6 +114,18 @@ class ModelArray {
      * Construct an unnamed ModelArray of Type::H
      */
     ModelArray();
+    /*!
+     * @brief Construct a ModelArray of the given type and name
+     *
+     * @param t The ModelArray::Type for the new object.
+     * @param bounds The physical lower and upper bounds for the new object. See lowerPhysicalLimit
+     * and upperPhysicalLimit.
+     */
+    ModelArray(const Type t, const std::pair<double, double>& bounds)
+        : ModelArray(t)
+    {
+        setLimits(bounds.first, bounds.second);
+    }
```

Comment:
Suggest keeping a constructor with parameters `ModelArray(const Type t)`, which delegates to this constructor by supplying default limit arguments of `{-std::numeric_limits<double>::max(), std::numeric_limits<double>::max()}` (not infinity).

Reply:
I'm not sure how that would work, because this constructor calls the ``ModelArray(const Type type)`` constructor. But using a default value is a good idea, so I removed this constructor and added a bounds argument to the existing ``ModelArray(const Type type)`` constructor, with ``{-std::numeric_limits<double>::max(), std::numeric_limits<double>::max()}`` as default.